ui/components/ReactSelectWrapper.js

```diff
@@ -125,12 +127,13 @@ const components = {
   Placeholder,
   SingleValue,
   ValueContainer,
+
 };
 
 class ReactSelectWrapper extends React.Component {
   render() {
     const {
-      classes, theme, label, placeholder, onChange, onInputChange, value, options, error, noOptionsMessage = "No Options"
+      classes, theme, label, placeholder, onChange, onInputChange, value, options, error, noOptionsMessage, style = "No Options"
```

Comment:
style="NoOptions"?? Oh, you have edited the code wrong way.
```suggestion
      classes, theme, label, placeholder, onChange, onInputChange, value, options, error, noOptionsMessage= "No Options", style={}
```

Reply:
Were you able to setup the development server properly? I see you had problems in the past.

Reply:
Yes, @Abhishek-kumar09   I have facing a problem related to setup development server (can't able to see the changes which I have made in localhost 3000)

Reply:
Please discuss such issues if you are facing in new-comers call or development call. You can't do development without your dev-server running...

Reply:
It's good for long term

Reply:
Sure